validator.go

```diff
@@ -142,5 +142,5 @@ func isValidWeekDay(val string, last int, ref time.Time) (bool, error) {
 		return false, nil
 	}
 
-	return ref.Day()/7 == nth-1, nil
+	return (ref.Day()-1)/7 == nth-1, nil
```

Comment:
strange this change still has all tests passed

Reply:
I think this is an edge case that wasn't covered in the previous unit tests. If we use a cron string that does first/second/etc monday/tuesday/etc of the month, then we get in a situation where days 1-6 are in week 0, but day 7 is week 1. This change makes it that days 1-7 are in week 0.